bridge_adaptivity/module/models.py

```diff
@@ -126,7 +127,9 @@ def sequence_ui_details(self):
                 )
             elif ui_option == CollectionOrder.OPTIONS[1][0]:
                 grade = self.collection_order.grading_policy.calculate_grade(self)
-                details = f"{CollectionOrder.OPTIONS[1][1]}: {grade}"
+                if isinstance(grade, float):
+                    grade = round(grade * 100, 1)
```

Comment:
It seems we've already cut useless part of the number, why do we need round here?